lib/std/io/reader.zig

```diff
@@ -60,12 +60,7 @@ pub fn Reader(
             return self.readAllArrayListAligned(null, array_list, max_append_size);
         }
 
-        pub fn readAllArrayListAligned(
-            self: Self,
-            comptime alignment: ?u29,
-            array_list: *std.ArrayListAligned(u8, alignment),
-            max_append_size: usize
-        ) !void {
+        pub fn readAllArrayListAligned(self: Self, comptime alignment: ?u29, array_list: *std.ArrayListAligned(u8, alignment), max_append_size: usize) !void {
```

Comment:
Ugh, I think this was OK having the arguments broken over a couple of lines. To force/convince `zig fmt` to have the arguments on multiple lines is to have *every* argument delimited by a comma, including the very last argument. So doing this:

```suggestion
        pub fn readAllArrayListAligned(self: Self, comptime alignment: ?u29, array_list: *std.ArrayListAligned(u8, alignment), max_append_size: usize,) !void {
```

should convince the `zig fmt` to leave the previous formatting alone.